File: Python/11-most-common-letter.py
def most_common_letter(str):
    letter = ""
    letter_count = 1

    for i in range(0, len(str)):
        count = 0
        for j in range(i, len(str)):
            if str[i] == str[j]:
                count += 1
        if count > letter_count:
            letter_count = count
            letter = str[i]

    return [letter, letter_count]


print(most_common_letter("aboo"))
print(most_common_letter("abeusessu"))

File: Javascript/11-most-common-letter.js
function mostCommonLetter(string) {
	var lettersChecked = "";
	var letter = "";
	var count = 0;

	for (var i = 0; i < string.length; i++) {
		var curLetter = string[i];
		var curCount = 1;

		for (var j = 0; j < string.length; j++) {
			if (i == j) {
				continue;
			} else if (curLetter == string[j]) {
				curCount++;
			}
		}

		if (curCount > count) {
			letter = curLetter;
			count = curCount;
		}
	}

	return [letter,count];
}

console.log(mostCommonLetter("accceuoaaa"));